Validators/CheckinCheckoutValidator.py

```diff
@@ -0,0 +1,34 @@
+from datetime import date
```

Comment:
file names should follow the snake_case convention, so it should be `validators/checkin_checkout_validator.py`